sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2024-2025.bpmn

```diff
@@ -72,1352 +216,1475 @@ dmnFullTimeX (comes from dmnFullTime)</bpmn:text>
+          <zeebe:output source="=0" target="calculatedDataTotalEligibleDependants" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0x46mhe</bpmn:incoming>
-      <bpmn:outgoing>Flow_1qeb5i1</bpmn:outgoing>
+      <bpmn:incoming>Flow_1li9w29</bpmn:incoming>
+      <bpmn:outgoing>Flow_1qvqrur</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_1p23tfj" name="awardNetCSLFPotentialAmount">
+    <bpmn:intermediateThrowEvent id="Event_1kdusuh" name="Total Parent Eligible Dependants">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=min(&#10;  (dmnFullTimeAwardFederalAllowableLimits.limitAwardCSLFWeeklyRate * offeringWeeks),&#10;  (calculatedDataFederalAssessedNeed - federalAwardNetBCSLPotentialAmount)&#10;)" target="federalAwardNetCSLFPotentialAmount" />
+          <zeebe:output source="=max(calculatedDataTotalEligibleParent1Dependants,calculatedDataTotalEligibleParent2Dependants)" target="calculatedDataTotalParentEligibleDependants" />
+          <zeebe:output source="=max(calculatedDataTotalEligibleParent1ContributionDependants,calculatedDataTotalEligibleParent2ContributionDependants)" target="calculatedDataTotalParentEligibleContributionDependants" />
+          <zeebe:output source="=0" target="calculatedDataTotalEligibleDependants" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_1qeb5i1</bpmn:incoming>
-      <bpmn:outgoing>Flow_01f8hwa</bpmn:outgoing>
+      <bpmn:incoming>Flow_0ovh763</bpmn:incoming>
+      <bpmn:outgoing>Flow_119pdg5</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_160re05" name="awardNetCSLFAmount">
+    <bpmn:intermediateThrowEvent id="Event_0l38vd4" name="Total Parent Net Income">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=sum(federalAwardNetCSGPAmount,federalAwardNetCSGDAmount,federalAwardNetCSGFAmount,federalAwardNetCSGTAmount)" target="federalAwardNetCSGAmount" />
-          <zeebe:output source="=0.6 * (calculatedDataFederalAssessedNeed - federalAwardNetBCSLPotentialAmount - federalAwardNetCSLFPotentialAmount)" target="federalAwardFederalUnmetNeed" />
-          <zeebe:output source="=max(0,min(federalAwardNetCSLFPotentialAmount,max(0,(federalAwardNetCSLFPotentialAmount + federalAwardFederalUnmetNeed - federalAwardNetCSGAmount))))" target="federalAwardNetCSLFAmount" />
+          <zeebe:output source="=min(dmnFullTimeCalculatedDataProgramYearMaximums.limitParentCPPAmount, (parent1CppEmployment + parent1CppSelfemploymentOther)) + min(dmnFullTimeCalculatedDataProgramYearMaximums.limitParentEIAmount,parent1Ei) " target="calculatedDataParent1IncomeDeductions" />
```

Comment:
Correct me if wrong, the dmn lookup `dmnFullTimeCalculatedDataProgramYearMaximums` seems to be wrong. 

I checked the file attached my @CarlyCotton and it seems to have the right name. 

![image](https://github.com/user-attachments/assets/07b9ed32-c933-42a6-9137-05fb21735ddb)

Reply:
I got the reason. Never mind. I see that the result variable of dmn lookup was `dmnFullTimeCalculatedDataProgramYearMaximums`. 

Thanks for adjusting it.